### PR TITLE
Remove set as startup task button for non-event activities

### LINKED ISSUE #16345: Task-type Workflows Activities an also be set as startup task
### Describe the bug

You can set Task-type Activities as startup tasks on the workflow canvas. However, this doesn't make sense, since a Task can't start a workflow, only an Event can.

### Orchard Core version

22fbbb491068316a71b2feff7014dd828a73a654

### To Reproduce
1. Add any Task like Notify to a workflow.
2. Notice that you can set it as a startup task.

### Expected behavior
The startup task button only appears for events. Also, the terminology "startup _task_" is incorrect as well, since it should say "startup _event_".

### Logs and screenshots

E.g. a Notify Task can't start a workflow, but you can set it as a startup task:

image

### PR DESCRIPTION
**Pull request by mdameer**
Fix #16345 

Remove set as startup task button for non-event activities
Rename "Startup task" tooltip to "Startup event"

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Workflows/Views/Activity.Design.cshtml [hunk 1]
```diff
@@ -24,7 +24,10 @@
         </div>
     }
     <div class="activity-commands">
-        <a class="btn btn-sm btn-secondary activity-start-action" href="#" data-bs-toggle="button" title="@T["Startup task"]"><i class="fa-solid fa-power-off" aria-hidden="true"></i></a>
+        @if (isEvent)
+        {
+            <a class="btn btn-sm btn-secondary activity-start-action" href="#" data-bs-toggle="button" title="@T["Startup event"]"><i class="fa-solid fa-power-off" aria-hidden="true"></i></a>
+        }
```
Comment (hishamco) [inline src/OrchardCore.Modules/OrchardCore.Workflows/Views/Activity.Design.cshtml:30]:
```suggestion
        }

```

**Review by hishamco — APPROVED** [review]

**Comment by hishamco** [comment]
I already approved the PR :)

@Piedone can we merge this

**Review by hishamco — APPROVED** [review]

**Review by Piedone — APPROVED** [review]
Looks good, thank you. But let's wait for the triage meeting later today to validate the issue in the first place :). Then I'll immediately merge if OK.

**Comment by Piedone** [comment]
We didn't have a meeting this week, so let's see again next week.

BTW you don't need to continuously merge from `main` (unless there's a new merge conflict).

**Comment by hishamco** [comment]
> We didn't have a meeting this week, so let's see again next week.

Is Seb ok?

**Comment by Piedone** [comment]
Too much waiting :).

**Pull request merged**